### PR TITLE
Freshness work for the overview article

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by msewaweru** [comment]
@CelesteDG , I have done a freshness pass for the overview article. Is it ok to point out that the cmdlets in this module are autogenerated as I have added in the first paragraph? Thanks.

File: microsoftgraph/docs-conceptual/overview.md [hunk 1]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
-- **Cross-platform support and available on PowerShell 7 and above**: Microsoft Graph PowerShell module works with PowerShell 7 and later and on all platforms including Windows, macOS, and Linux. It's also compatible with Windows PowerShell 5.1.
-- **Supports modern authentication**: Microsoft Graph PowerShell supports the Microsoft Authentication Library (MSAL) which offers more security that its Azure AD PowerShell Active Directory Authentication Library (ADAL) counterpart.
+- **Access to all Microsoft Graph APIs**: Microsoft Graph PowerShell is based on Microsoft Graph API. The Microsoft Graph API includes, in addition to Azure AD, APIs from other Microsoft services like SharePoint, Exchange, and Outlook, all accessed through a single endpoint with a single access token.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:24]:
```suggestion
- **Access to all Microsoft Graph APIs**: Microsoft Graph PowerShell is based on Microsoft Graph API. In addition to Azure AD, the Microsoft Graph API includes APIs from other Microsoft services like SharePoint, Exchange, and Outlook, all accessed through a single endpoint with a single access token.
```

File: microsoftgraph/docs-conceptual/overview.md [hunk 2]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
+- **Supports PowerShell 7**: Microsoft Graph PowerShell module works with PowerShell 7 and later. It's also compatible with Windows PowerShell 5.1.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:25]: Do we need the work module here? Or should just be "Microsoft Graph PowerShell works with..." it sounds awkward with module added like you meant to specify a particular module but forgot.

File: microsoftgraph/docs-conceptual/overview.md [hunk 3]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
+- **Cross-platform support**: Microsoft Graph PowerShell module works on all platforms including Windows, macOS, and Linux.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:26]:
```suggestion
- **Cross-platform support**: Microsoft Graph PowerShell works on all platforms including Windows, macOS, and Linux.
```

File: microsoftgraph/docs-conceptual/overview.md [hunk 4]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
+- **Supports modern authentication**: Microsoft Graph PowerShell supports the Microsoft Authentication Library (MSAL) which offers more security. For example, you can use Passwordless sign-in experiences.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:27]:
```suggestion
- **Supports modern authentication**: Microsoft Graph PowerShell supports the Microsoft Authentication Library (MSAL) which offers more security. For example, you can use passwordless sign-in experiences.
```

File: microsoftgraph/docs-conceptual/overview.md [hunk 5]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
+- **Supports external identities**: Users from other Azure AD tenants can authenticate to services in your tenant with Microsoft Graph PowerShell.
+- **Uses least privilege**: Microsoft Graph PowerShell permissions are NOT pre-authorized and users must perform one-time request for app permissions depending on their needs.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:29]:
```suggestion
- **Uses least privilege**: Microsoft Graph PowerShell permissions are not pre-authorized and users must perform one-time request for app permissions depending on their needs.
```

File: microsoftgraph/docs-conceptual/overview.md [hunk 6]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
+- **Advanced queries**: Microsoft Graph PowerShell supports rich, advanced queries via eventual consistency. For example, you can get a near-instant count of all users using advanced queries.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:30]: What is eventual consistency? Is that a term that we introduced in the past or is there an article where that's explained clearly/more thoroughly (and we should link to)?

File: microsoftgraph/docs-conceptual/overview.md [hunk 7]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
...
 - **Open source**: Feature teams and the community can create great PowerShell experiences and share them with everyone.
+- **Receives regular updates**: Microsoft Graph PowerShell commands are updated regularly to support the latest Graph API updates.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:32]: Do we have a cadence where we can mention what regularly means? Or if we can link to a changelog or what's new topic to tell them where to check out the latest and greatest that would be helpful.

**Review by CelesteDG — CHANGES_REQUESTED** [review]
@msewaweru Please see my comments.

File: microsoftgraph/docs-conceptual/overview.md [hunk 8]
```diff
@@ -15,16 +13,23 @@ ms.reviewer: maisarissi
 The Microsoft Graph PowerShell SDK acts as an API wrapper for the Microsoft Graph APIs, exposing the entire API set for use in PowerShell. It contains a set of cmdlets that helps you manage identities at scale from automating tasks to managing users in bulk using Azure Active Directory (Azure AD).
 It will help administer every Azure AD feature that has an API in Microsoft Graph.
 
-The Microsoft Graph PowerShell SDK is the replacement for the Azure AD PowerShell module and is recommended for interacting with Azure AD.
+The commands in Microsoft Graph PowerShell are autogenerated from the Microsoft Graph API schema making it easier to get faster updates and functionality. The cmdlet reference content is also autogenerated from the API reference.
```
Comment (CelesteDG) [inline microsoftgraph/docs-conceptual/overview.md:16]: This is fine. I like the transparency and additional info/context for the customers.

File: microsoftgraph/docs-conceptual/overview.md (hunk 7 above)
Comment (msewaweru) [reply microsoftgraph/docs-conceptual/overview.md:32]: Releases are currently done monthly. I will see if I can get a link to a changelog.

**Comment by msewaweru** [comment]
@CelesteDG, thanks for your review.

@timayabi2020, please merge this.

**Comment by msewaweru** [comment]
@CelesteDG , this is ready for merging. Thanks.

**Pull request merged**